.github/workflows/scheduled.yml

```diff
@@ -274,27 +399,63 @@ jobs:
           path: |
             /tmp/spark_aggregate_fuzzer_repro
 
-  linux-spark-fuzzer-run:
-    name: "Spark Fuzzer"
+  spark-bias-fuzzer:
+    name: Spark Bias Fuzzer
     runs-on: ubuntu-latest
+    container: ghcr.io/facebookincubator/velox-dev:centos8
     needs: compile
+    if: ${{ needs.compile.outputs.spark_bias  == 'true' }}
     timeout-minutes: 120
     steps:
 
-      - name: "Checkout Repo"
-        uses: actions/checkout@v4
+      - name: Download spark expression fuzzer
+        uses: actions/download-artifact@v4
         with:
-          ref: "${{ inputs.ref }}"
+          name: spark_expression_fuzzer
+
+      - name: Download Signatures
+        uses: actions/download-artifact@v4
+        with:
+          name: signatures
+          path: /tmp/signatures
+
+      - name: Run Spark Expression Fuzzer
+        run: |
+          ls /tmp/signatures
+          mkdir -p /tmp/spark_fuzzer_repro/
+            chmod -R 777 /tmp/spark_fuzzer_repro
+            chmod +x spark_expression_fuzzer_test
+            ./spark_expression_fuzzer_test \
+                --seed ${RANDOM} \
+                --duration_sec $DURATION \
+                --logtostderr=1 \
+                --minloglevel=1 \
+                --assign_function_tickets  $(cat /tmp/signatures/spark_bias_functions) \
+                --repro_persist_path=/tmp/spark_fuzzer_repro \
+            && echo -e "\n\nSpark Fuzzer run finished successfully."
 
-      - name: "Install dependencies"
-        run: source ./scripts/setup-ubuntu.sh
+      - name: Archive Spark expression production artifacts
+        if: ${{ !cancelled() }}
+        uses: actions/upload-artifact@v4
+        with:
+          name: spark-fuzzer-failure-artifacts
+          path: |
+            /tmp/spark_bias_fuzzer_repro
+
+  spark-fuzzer:
```

Comment:
Do we run the biased fuzzer and the normal fuzzer when we detect a change or only one of the two ? if we detect a change we need only run the biased fuzzer imv.

Reply:
yeah we run both but that's an easy fix